feat(trace-suspect-tags): Adding floating trigger on box selection end

Abdkhan14 opened the pull request:
Under the flag: `performance-spans-suspect-attributes`

https://github.com/user-attachments/assets/ae477636-e815-49f2-9e88-711160968fcf

Commit:
feat(trace-suspect-tags): Adding floating trigger on box selection end

Commit:
feat(trace-suspect-tags): Minor iteration

Commit:
feat(trace-suspect-tags): Minor iteration

edwardgou-sentry commented on static/app/views/explore/hooks/useChartBoxSelect.tsx:
```diff
@@ -41,13 +37,19 @@ export const EXPLORE_CHART_BRUSH_OPTION: BrushComponentOption = {
   const [brushArea, setBrushArea] = useState<EchartBrushAreas | null>(null);
+  const [pageCoords, setPageCoords] = useState<{x: number; y: number} | null>(null);
+
+  const [forceReActivateSelection, reActivateSelection] = useReducer(
```
might be worth leaving a comment on how some of these hooks work/are used.

edwardgou-sentry reviewed (approved):
lgtm, just a comment suggestion

Commit:
feat(trace-suspect-tags): Adding comments

Abdkhan14 committed:
feat(trace-suspect-tags): Implementing zoom in (#94215)

<img width="789" alt="Screenshot 2025-06-24 at 2 18 20 PM"
src="https://github.com/user-attachments/assets/1efbc0b4-571f-49fb-ae39-6241b5eae93b"
/>

---------

Co-authored-by: Abdullah Khan <abdullahkhan@PG9Y57YDXQ.local>
